axopy/pipeline/common.py

```diff
@@ -404,15 +404,40 @@ class Estimator(Block):
     estimator : object
         An object implementing the scikit-learn Estimator interface (i.e.
         implementing ``fit`` and ``predict`` methods).
+    return_proba : boolean, optional (default: False)
+        If True, return probability estimates.
+    return_log_proba : boolean, optional (default: False)
+        If True, return log of probability estimates.
     """
 
-    def __init__(self, estimator):
+    def __init__(self, estimator, return_proba=False):
```

Comment:
Missing the actual `return_log_proba` keyword argument :)

Reply:
How did I manage to miss that, lol! Well spotted!